exonum/src/runtime/rust/interfaces.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Important interservice communication interfaces.
+
+use crate::runtime::{dispatcher::Error as DispatcherError, MethodId};
+
+use super::{ExecutionError, Interface, TransactionContext};
+
+pub const INITIALIZE_METHOD_ID: MethodId = 0;
+
+/// This trait describes a service interface to pass initial configuration parameters into
+/// the started service instance.
+pub trait Initialize {
+    /// Initialize a service instance with the given parameters.
+    ///
+    /// The configuration parameters passed to the method are discarded immediately.
```

Comment:
`... are not saved by the Dispatcher`. `discarded` means that they are not used in method.

Reply:
@MakarovS Help please.

Reply:
I believe you wanted to mention @Maria-Nosyk 
But I agree with @vitvakatu's suggestion btw

Reply:
But in fact it means that parameters exist only in RAM and after initialize dissapear from it.